src/paper_feeds/app.py

```diff
@@ -58,6 +63,8 @@ async def search(request: Request, search: Annotated[str, Form()]):
 @app.get('/journals/{issn}')
 async def journal_page(request: Request, issn:str):
     journal = crossref.load_journal(issn)
+    if journal.homepage_url is None:
+        journal.homepage_url = '-'
```

Comment:
re: my comment below making the field optional in presentation, no strong preference between displaying null value like this vs. just not displaying at all - we should make these endpoints return JSON when expected so this isn't a problem in case someone wants computer-readable access to this API at some point.